lab-justin/lib/lab03.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = function(cb){
+  const fs = require('fs');
+  const filePaths = [
```

Comment:
These should really be passed into this function, not hardcoded. The filepaths should be in the index and be passed to this as an argument.